packages/perseus/src/interactive2/wrapped-ellipse.tsx

```diff
@@ -53,6 +58,21 @@ class WrappedEllipse extends WrappedDrawing {
             // movable wrapper so that when moved the browser does not scroll page
             this.wrapper.style.touchAction = "none";
 
+            // Option 2: instead of addToMouseLayerWrapper(this.wrapper)
+            // we could do addToMouseLayerWrapper(reactRoot)
+            reactRender(
```

Comment:
Can we use a React portal instead?  `reactRender` is an abstraction over `ReactDOM.render()` that I think we should remove eventually as React Context's don't flow through it whereas they do with Portals.